app/controllers/users/sessions_controller.rb

```diff
@@ -16,6 +16,10 @@ def new
   # POST /resource/sign_in
   def create
     super
+    role = current_user.roles.find { |r| r.name == :super_admin }
+    role ||= current_user.roles.find { |r| r.name == :org_user }
```

Comment:
Is this missing the `org_admin` role?

Reply:
`current_role` is just used to figure out which view to show users (e.g. which dashboard page). There isn't any difference between `org_user` and `org_admin` in terms of views, so I left it out. Having said that, I can see this being confusing, so I'll add it in.